### PR TITLE
New package: Homebrew.Cortexso version 0.5.0

### PR DESCRIPTION
**Pull request by jan-service-account**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/165191)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-07-31 14:47:24.867 [FAIL] Installer failed security check. Url: https://github.com/janhq/cortex/releases/download/v0.5.0/cortex-installer-0.5.0-amd64-windows.exe Result: 0x80004005

(Automated response - build 887.)

File: manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Homebrew.Cortexso
```
Comment (Trenly) [inline manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.installer.yaml:4]:
```suggestion
PackageIdentifier: janhq.HomebrewCortexso
```
Standard practice for GitHub releases is `publisher.repo`

File: manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Homebrew.Cortexso
```
Comment (Trenly) [inline manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: janhq.HomebrewCortexso
```
Standard practice for GitHub releases is `publisher.repo`

File: manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Homebrew.Cortexso
```
Comment (Trenly) [inline manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.yaml:4]:
```suggestion
PackageIdentifier: janhq.HomebrewCortexso
```
Standard practice for GitHub releases is `publisher.repo`

File: manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Homebrew.Cortexso
+PackageVersion: 0.5.0
+PackageLocale: en-US
+Publisher: Homebrew
```
Comment (Trenly) [inline manifests/h/Homebrew/Cortexso/0.5.0/Homebrew.Cortexso.locale.en-US.yaml:7]:
```suggestion
Publisher: janhq
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

https://www.virustotal.com/gui/file/da240af392dff62221f3c1595ea97cf1325e090fb9345b0d9724ba775fda8713

**Pull request closed without merging**